Form-Controls/index.html

```diff
@@ -13,6 +13,37 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
+        <div>
+          <label for="name">Name</label>
+          <input type="name" name="name" id="name" minlength="2" required />
```

Comment:
I can see you've used a `minlength` property here with a reasonable value! Good stuff.